easybuild/easyconfigs/c/CMake/CMake-3.12.1-GCCcore-7.3.0.eb

```diff
@@ -31,7 +31,9 @@ osdependencies = [
     ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
 ]
 
-configopts = '-- -DCMAKE_USE_OPENSSL=1 -DCMAKE_PREFIX_PATH=$EBROOTNCURSES'
+configopts = '-- -DCMAKE_USE_OPENSSL=1'
+configopts += ' -DCURSES_CURSES_LIBRARY=$EBROOTNCURSES/lib/libcurses.%s' % SHLIB_EXT
+configopts += ' -DCURSES_NCURSES_LIBRARY=$EBROOTNCURSES/lib/libncurses.%s' % SHLIB_EXT
```

Comment:
@smoors We need to specify *both*? Also, shouldn't we leave the `CMAKE_PREFIX_PATH` in place?

Maybe I'm too overly careful here, but I just want to avoid introducing regressions.

Also, it's still not clear to me *why* it didn't work for you without the `-DCURSES_*`, it would be good to know...